wiki_docs/Create How_GTM_works

```diff
@@ -0,0 +1,47 @@
+User can add or manage the AdWords, Google analytics, Flood-light and non-Google tags using the GTM.
+
+How it works
+  Followings are main components of the GTM:
+
+Account
```

Comment:
An account is where you will hold all of the containers for tracking tags. A company or organization has an account, which can hold GTM for multiple sites or apps associated with it.